tests/tensorflow/test_callbacks.py

```diff
@@ -27,52 +27,37 @@
 
 REF_CKPT_DIR = {
     'epoch': ['checkpoint',
-              'ckpt-1.data-00000-of-00002',
```

Comment:
Checkpoint logic is changed in TF 2.10. This somehow leads to a different number of shards. For example, `MultiDeviceSaver` class which performs the actual saving is implemented differently in 2.10. Compare:
- Original: https://github.com/tensorflow/tensorflow/blob/r2.9/tensorflow/python/training/saving/functional_saver.py
- In 2.10 is nor longer present in original location https://github.com/tensorflow/tensorflow/blob/r2.10/tensorflow/python/training/saving/functional_saver.py and moved here https://github.com/tensorflow/tensorflow/blob/r2.10/tensorflow/python/checkpoint/functional_saver.py with the change of implementation